rasa/core/policies/rule_policy.py

```diff
@@ -853,7 +853,9 @@ def predict_action_probabilities(
                 None,
             )
 
-        rules_action_name, source = self._find_action_from_rules(tracker, domain)
+        rules_action_name, source = self._find_action_from_rules(
+            tracker, domain, use_text_for_last_user_input=False
+        )
```

Comment:
@Ghostvv I merged master into `e2e` and got some merge conflicts. I think I messed up something here. Also one test is failing that should be related to that. Can you please take a look?

Reply:
This was the merge commit: https://github.com/RasaHQ/rasa/commit/3ed6d0421d47a74244ebe0a9ab76aeb07c713703

Reply:
pushed to e2e and merged e2e here